### PR TITLE
Enable rpm build

### PR DESCRIPTION
**Pull request by marun**

### CODE REVIEW

**Comment by marun** [comment]
I'm isolating the origin build machinery in `./origin-build` to simplify carrying, so it will be necessary to update the [ci config](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/kubernetes/openshift-kubernetes-marun-testing.yaml) accordingly.

File: origin-build/images/openldap/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,23 @@
+FROM openshift/openldap-2441-centos7:latest
```
Comment (sttts) [inline origin-build/images/openldap/Dockerfile:1]: openldap?  👀

Comment (marun) [reply origin-build/images/openldap/Dockerfile]: Removed.

**Comment by marun** [comment]
Merging to enable rehearsal testing via https://github.com/openshift/release/pull/9448

**Pull request merged**